### PR TITLE
Added redis selected database

### PR DESCRIPTION
**Pull request by kjdev**
Phalcon\Cache\Backend\Redis selected database.

``` php
new Phalcon\Cache\Backend\Redis($frontCache, array(
  'host' => 'localhost',
  'port' => 6379,
  'index' => 1  // Added index option (database number)
));
```

### CODE REVIEW

**Comment by Green-Cat** [comment]
:+1:

File: phalcon/cache/backend/redis.zep [hunk 1]
```diff
@@ -122,6 +122,14 @@ class Redis extends Backend implements BackendInterface
 			}
 		}
 
+		if fetch index, options["index"] {
+			let success = redis->select(index);
+
+			if !success {
+				throw new Exception("Redisd server selected database failed");
```
Comment (sergeyklay) [inline phalcon/cache/backend/redis.zep]: By default Redis use index 0. It would be nice to have a value by default here

**Comment by kjdev** [comment]
Pull Request but was closed, whether this proposal would be rejected?

**Comment by sergeyklay** [comment]
@kjdev It seems need another PR ))

**Comment by kjdev** [comment]
Thanks.
I will issue another PR.

**Pull request closed without merging**